.github/workflows/test.yml

```diff
@@ -0,0 +1,105 @@
+name: Test SkyPortal
+
+on: [push]
+
+jobs:
+  test:
+    name: Test SkyPortal
+    runs-on: ubuntu-latest
+
+    services:
+      postgres:
+        image: postgres:11.8
```

Comment:
I thought we're on postgres 12.x -- please confirm.

Reply:
I'm now using postgres latest